### DISCUSSION TITLE
Does the new version of prisma (v 2) support MongoDB?

### DISCUSSION THREAD

**Discussion by kisna72** [prisma/prisma#2364] [Q&A] [answered]
Asking becuase I looked up the docs and didn't see any reference to it.

**Comment by ryands17** [comment, marked as answer]
Hi @kisna72 :wave: 
Currently MongoDB is not supported by Prisma 2 but will be in the future. You can track the progress of the feature request [here](https://github.com/prisma/prisma/issues/1277)

**Reply by kisna72** [reply]
Thanks for the response.. :)

**Comment by ludralph** [comment]
Hi @kisna72,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
